Source/DataValidationExtensions/Public/DVEDataValidator.h

```diff
@@ -216,6 +216,48 @@ struct DATAVALIDATIONEXTENSIONS_API FDVEDataValidator
         return *this;
     }
 
+    template < typename _KEY_TYPE_, typename _VALUE_TYPE_ >
+    FDVEDataValidator & MapHasNoNullKey( const FName property_name, const TMap< _KEY_TYPE_, _VALUE_TYPE_ > & map )
+    {
+        TArray< _KEY_TYPE_ > keys;
+        map.GenerateKeyArray( keys );
+
+        for ( const auto * key : keys )
+        {
+            if ( key == nullptr )

```

Comment:
use IsValueNull (defined below) to make that code work also for types that have a `.IsNull` function